.github/workflows/test-coverage.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         database-type: [file, memory]
         mutation-type: [gql, collection-named, collection-save]
 
-    runs-on: ubuntu-latest
+    runs-on: runs-on=${{ github.run_id }}-${{ github.run_attempt }}-${{ strategy.job-index }}/spot=pco/cpu=16+32/family=m7*+c7*/extras=s3-cache
```

Comment:
question: why not also other `runs-on` as well?

Reply:
Looks like it would be sensible to explore that in another chunk of work if it is not straight forward to do so right now.  This is a very good improvement to get in in itself, and I think Fred mentioned a problem/hassle with the mac runner.

Reply:
I'll add it to the other linux based jobs :)

Reply:
Thanks Fred!

Reply:
@shahzadlone, Change has been made.